src/components/shared/SearchBar/SearchBar.tsx

```diff
@@ -43,15 +44,24 @@ const SearchBar: React.FC<searchBarProps> = ({ onSearch, page }) => {
     setLoading(true);
     try {
       logger.debug(`Fetching search results for query: "${query}"`);
-      const data = await fetchSellers(undefined, undefined, query);
+      
```

Comment:
Hi @DarinHajou - have you had a chance to test this section?  It seems that the `mapInstance` is consistently null, which is causing the bounds error **[screenshot attached below]**.  As a result, the submit search functionality isn't quite working as it did before.  When you have a moment, could you and/ or @joey364 please take a look at this? 
 Thanks!
![image](https://github.com/user-attachments/assets/3e7da11d-9441-47c0-8e4f-220b6dbf5f14)

Reply:
Hi again @DarinHajou - 
After further testing, it seems we are still triggering the fetching of sellers with even slight map movements, which is placing quite a load on the fetchSellers API endpoint.  I’m not sure how this improves performance compared to the origin/radius approach we had earlier, where we could control additional fetching.  Do we have the same level of control with the bounding box approach?  I might be missing something here—please let me know if I am.

Also, it looks like we're fetching a maximum of 36 sellers, even without performing a search.  It seems 36 is the limit we can display on the map, rather than pulling all sellers within the bounds.  You may want to confirm with our PM, but I believe the 36-limit + [review count + recent reviews] applies to search results only.  We should still display all  sellers even when no search is submitted.

Reply:
> Also, it looks like we're fetching a maximum of 36 sellers, even without performing a search. It seems 36 is the limit we can display on the map, rather than pulling all sellers within the bounds. You may want to confirm with our PM, but I believe the 36-limit + [review count + recent reviews] applies to search results only. We should still display all sellers even when no search is submitted.

Please disregard this comment @DarinHajou.  Our PM confirmed to keep the limit even with no searches.  Thx!